templates/resources/publishing-a-data-package.md

```diff
@@ -18,8 +18,6 @@ The process of publishing via the [EDI Data Portal](https://portal.edirepository
 
 From the Data Portal, click the **Login** button and enter EDI account credentials.
 
-<img class="screen-shot" src="/static/images/login-portal.png" width="80%">
```

Comment:
@kzollove Don't remove images from this page.

Reply:
Images have been restored in the PR